### DISCUSSION TITLE
Hooks for Backups API

### DISCUSSION THREAD

**Discussion by AfdhalAtiffTan** [pocketbase/pocketbase#5069] [Q&A] [answered]
Hello,

I am trying to remove `logs.db` from the generated backup file in `pb_data/backups` via `/api/backups` endpoint.

I am using `zip -d pb_backup_app_20240612110722.zip logs.db`.

I assume the the generated `pb_backup_app_20240612110722.zip.attrs` can be ignored, is this correct?

If so, is it possible to have a hook like `onAfterBackup` or `onBeforeRestore` or `onAfterUpload`? 
I do wish to verify the file signature in the said hooks.

If not, is there any workaround?

**Comment by ganigeorgiev** [comment, marked as answer]
There is no option to disable only the logs from the backup at the moment.

Backup file hooks exists in my todo and will be added in the future but there are no ETAs. The backup implementation in general also will slightly change in attempt to minimize the write-lock time.

**Reply by AfdhalAtiffTan** [reply]
Thank you for the super fast reply.
